.goreleaser.yaml

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+env:
+  # Set env var TAG_OVERRIDE to override the container images' tag.
+  # By default we use Version as the tag.
+  - TAG={{ if index .Env "TAG_OVERRIDE"  }}{{ .Env.TAG_OVERRIDE }}{{ else }}'{{ .Version }}'{{ end }}
```

Comment:
Why quote version but not `TAG_OVERRIDE`?

Reply:
This is now removed. 

For one, it didn't actually work (my bad). At this stage, `{{ .Version }}` only yields an empty string. Per goreleaser doc, the built-in templates only works in `*_template` (e.g. `name_template`) blocks.
For two, in the new flow we will have to `git tag` to release. So I'm simplifying it solely rely on the git tag as the image tag.